components/common/Nav.tsx

```diff
@@ -123,7 +126,8 @@ const NavBarContainer = ({ children, ...props }) => {
       justify="space-between"
       wrap="wrap"
       w="100%"
-      p={8}
+      py={4}
+      px={RESPONSIVE_PADDING}
```

Comment:
Main change here: vertical padding to 16px, horizontal padding to 16px (phone) or 32px (tablet/desktop), as per the design.